app/views/content/life-as-a-teacher/teaching-as-a-career/early-career-teachers.md

```diff
@@ -2,34 +2,29 @@
 title: "Early career teachers (ECTs)"
 heading: "Early career teachers (ECTs)"
 description: |-
-  Find out about the support early career teachers get when they start teaching, including mentor support and training based on the early career framework (ECF).
+  Find out about the support early career teachers get when they start teaching, including mentor support and training based on the initial teacher training early career framework (ITTECF).
 promo_content:
   - "content/shared/block-promos/mailing_adviser_routes"
 ---
 
 You’ll be known as an early career teacher (ECT) for your first 2 years in teaching. The term early career teacher (ECT) has replaced newly qualified teacher (NQT).
 
-During your 2 year induction, you’ll receive a package of training and support based on the [early career framework (ECF)](https://www.gov.uk/guidance/how-the-early-career-framework-ecf-supports-induction). This is designed to help you build the skills and knowledge you need to feel confident and empowered in your classroom.
+During your 2-year induction, you’ll receive a package of training and support based on the [initial teacher training and early career framework (ITTECF)](https://www.gov.uk/government/publications/initial-teacher-training-and-early-career-framework?). Your package of training and support is referred to as the early career teacher entitlement (ECTE).
```

Comment:
This link is to guidance for providers - doesn't seem appropriate?

Would this be better: https://www.gov.uk/government/publications/early-career-teacher-entitlement/early-career-teacher-entitlement
(need to check with CPD team)

Reply:
and then maybe the link text would be `early career teacher entitlement (ECTE)` at the end of the second sentence